### ISSUE TITLE
classesDir is deprecated

### ISSUE THREAD

**Issue by boris-petrov** [spotbugs/spotbugs#292]
Because of SpotBugs using `classesDir`, the following warning appears when using Gradle 4+:

```
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
```

Instead, `classesDirs` should be used.

**Comment by KengoTODA** [comment]
will be closed by #337

**Comment by KengoTODA** [comment]
Gradle Plugin v1.4 solved this issue
